cwms-data-api/src/main/webapp/rsql.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
```

Comment:
I'm not going to worry about it at the moment, and frankly this will be useful anyways, but we do now have the Read-The-Docs documentation setup in the docs folder.